src/libcore/tests/manually_drop.rs

```diff
@@ -21,4 +21,8 @@ fn smoke() {
 
     let x = ManuallyDrop::new(TypeWithDrop);
     drop(x);
+
+    // also test unsizing
+    let x : Box<ManuallyDrop<[TypeWithDrop]>> = Box::new(ManuallyDrop::new([TypeWithDrop]));
```

Comment:
This would be less confusing if the array had two elements or something.

Reply:
Done.